pkg/cvo/metrics.go

```diff
@@ -9,12 +9,15 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/client-go/tools/cache"
+	"k8s.io/klog"
 
 	configv1 "github.com/openshift/api/config/v1"
 	"github.com/openshift/cluster-version-operator/lib/resourcemerge"
 	"github.com/openshift/cluster-version-operator/pkg/internal"
 )
 
+const noChannel string = "NoChannel"
```

Comment:
If we're going to define a constant for this string, probably use it in the places that are currently hard-coding the string:

```console
$ git --no-pager grep NoChannel origin/pr/357 pkg 
origin/pr/357:pkg/cvo/availableupdates.go:                      Type: configv1.RetrievedUpdates, Status: configv1.ConditionFalse, Reason: "NoChannel",
origin/pr/357:pkg/cvo/cvo_test.go:                                      Reason:  "NoChannel",
origin/pr/357:pkg/cvo/metrics.go:const noChannel string = "NoChannel"
origin/pr/357:pkg/cvo/metrics.go:                       // Reset time if reason has tranisitioned from NoChannel to avoid alerting too quickly
```

Reply:
Obsolete as of 06c550f.